lessons/22-js-conditionals/index.js

```diff
@@ -0,0 +1,12 @@
+alert('Enter two numbers to see if the first is greater than the second');
+
+var firstNumber = prompt("enter the first number");
+var secondNumber=prompt("enter the second number");
+if (firstNumber>secondNumber ) {
+    message = 'firstNumberis greater';
+
+  } else if (firstNumber < secondNumber) {
+    message = 'firstNumberis lesser';
+  } else {
+    message = 'the numbers are the same';
+  }
```

Comment:
We haven't declared message here yet so this would technically be invalid.

We'd need to do:

```javascript
let message; // declares it, allows us to assign it a value later
if (firstNumber>secondNumber) {
  // ... the rest of your code
```